src/app/app-routing.module.ts

```diff
@@ -1,7 +1,10 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { DashboardComponent } from './dashboard/dashboard.component';
 
-const routes: Routes = [];
+const routes: Routes = [{
+  path: '',component: DashboardComponent
```

Comment:
the route can be /dashboard or something, I'll merge and we'll discuss later about the routes 👍